<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BD</class>
 <widget class="QMainWindow" name="BD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BD</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QTableWidget" name="outputTable"/>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLineEdit" name="orderIdLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="displayAllButton">
      <property name="text">
       <string>Show All Orders</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="displayUncompletedButton">
      <property name="text">
       <string>Show Uncompleted Orders</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="findButton">
      <property name="text">
       <string>Find Theme and Text</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>438</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
